### PR TITLE
Automate csv download

### LINKED ISSUE #10: Automating sheets download
Not issue exactly but a suggestion

On long run, it is advisable to automate pulling data from google sheet so that site can be updated more frequently.

Proposal:
Using gspread module, we can use google api to pull data rather manually download each and every 12 sheets.

Installation:
`pip install gspread`

Authentication:
Refer to this [documentation](https://gspread.readthedocs.io/en/latest/oauth2.html). I will put PR for README soon.

### PR DESCRIPTION
**Pull request by kzuri**
In reference to #10

### CODE REVIEW

**Comment by TheReconPilot** [comment]
After doing the setup according to the [documentation](https://gspread.readthedocs.io/en/latest/oauth2.html), the program runs (after correcting `df_temp` to `df`), but it overwrites the courses file with the instructor data. Would need to sort that out.

File: google-sheet-api.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#link_to_sheet = ""
+#sh = gc.open(link_to_sheet)
+
+worksheet_list = sh.worksheets()
+
+subjects = ['mth','hss','ecs','chm','phy','bio']
+header = ['courses','instructors']
+
+for i,worksheet in enumerate(worksheet_list):
+    df_temp = pd.DataFrame(worksheet.get_all_records())
```
Comment (TheReconPilot) [inline google-sheet-api.py:25]: To be corrected to df

File: google-sheet-api.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    
+    if i%2==0:
+        df.to_csv(os.path.join("data",subjects[i//2]+"_"+header[0]+".csv"))
+    else:
+        df.to_csv(os.path.join("data",subjects[i//2]+"_"+header[0]+".csv"))
```
Comment (TheReconPilot) [inline google-sheet-api.py]: For instructors, it would be corrected to header[1]. It works perfectly after that.

**Comment by kzuri** [comment]
> After doing the setup according to the [documentation](https://gspread.readthedocs.io/en/latest/oauth2.html), the program runs (after correcting `df_temp` to `df`), but it overwrites the courses file with the instructor data. Would need to sort that out.

Yup sorry for that. I think I pushed older version of file. I will quickly correct it. Yes, you are right df should be converted to df_temp and header error is caused due to copy paste mistake

Will push new file

**Review by adch99 — APPROVED** [review]

**Pull request merged**